sdk/src/test/java/io/dapr/client/DaprClientHttpAdapterTest.java

```diff
@@ -0,0 +1,671 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+package io.dapr.client;
```

Comment:
General comment not on this file: if you are using intellij can you please `Code` menu -> Reformat Code.  There is odd formatting.

Reply:
+1 there are too many return spaces between the lines.

Reply:
It is done. Can you check again please? Let me know if there is any other change i need to do.

Reply:
We are using 2 spaces, BTW.

Reply:
![image](https://user-images.githubusercontent.com/130954/72553863-687f5000-384e-11ea-99a4-2d10b07cd059.png)